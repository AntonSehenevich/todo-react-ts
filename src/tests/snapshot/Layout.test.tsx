import renderer from 'react-test-renderer'
import Layout from '../../components/layout'

test('renders correctly', () => {
  const tree = renderer.create(<Layout />).toJSON()

  expect(tree).toMatchSnapshot()
})
