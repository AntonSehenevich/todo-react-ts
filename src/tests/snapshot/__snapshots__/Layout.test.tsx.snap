// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="layout"
>
  <div
    className="content"
  />
</div>
`;
